src/main/kotlin/com/terraformation/backend/accelerator/db/ModuleEventStore.kt

```diff
@@ -74,9 +75,17 @@ class ModuleEventStore(
     val now = clock.instant()
     val user = currentUser()
 
+    val eventStatus =
+        when {
+          now.isBefore(startTime) -> EventStatus.NotStarted
+          now.isAfter(endTime) -> EventStatus.Ended
+          else -> EventStatus.InProgress
+        }
```

Comment:
If the "starting soon" status is intentionally missing here, a brief comment explaining why would be good. (Same in the update function.)

Reply:
Refactored to constant